src/views/article.vue

```diff
@@ -1,69 +1,100 @@
 <template>
-    <div class="article">
+    <div class="article" v-if="article">
         <!-- MAIN PARAGRAPH -->
         <div class="main-paragraph">
             <div class="main-paragraph-text-box">
-                <h2 class="main-paragraph-title">{{ wikiArticles.title }}</h2>
-                <p class="main-paragraph-text" v-html="wikiArticles.paragraph">{{ wikiArticles.paragraph }}</p>
+                <h2 class="main-paragraph-title">{{ article.title }}</h2>
+                <p class="main-paragraph-text" v-html="getParsedText(article.paragraph)"></p>
             </div>
-            <div class="main-paragraph-image-box" v-if="wikiArticles.image"
-                 :style="{ 'background-image': 'url(' + require('../assets/articles/' +  articleName + '/' + wikiArticles.image) + ')' }">
+            <div class="main-paragraph-image-box" v-if="article.image" :style="getImageStyle(article)">
                 <img class="image-box-deco desktop-only" src="../assets/groups/vector-for-blocks.svg" alt="">
-                <img class="image-box-deco mobile-only" src="../assets/mobile-image-deco.svg" alt="">
+                <img class="image-box-deco mobile-only" src="../assets/deco/mobile-image-deco.svg" alt="">
             </div>
         </div>
 
         <!-- SUB PARAGRAPHS -->
-        <div class="paragraph" v-for="(chapter, i) in wikiArticles.chapters" :key="chapter.index">
+        <div class="paragraph" v-for="(chapter, i) in article.chapters" :key="chapter.index">
             <!-- DIVIDER -->
             <img class="divider desktop-only" src="../assets/groups/vector-divider.svg" alt="">
             <img class="divider mobile-only" src="../assets/groups/divider-mobile.svg" alt="">
             <!-- PARAGRAPH TEXT AND IMAGE -->
             <div class="paragraph-content" :class="{'reverse-element': i%2 !== 0}">
-                <div class="image-box"
-                     :style="{ 'background-image': 'url(' + require('../assets/articles/' +  articleName + '/' + wikiArticles.chapters[i].image) + ')' }"
-                     :class="{'reverse-image': i%2 !== 0}">
+                <div class="image-box" v-if="article.chapters[i].image"
```

Comment:
I think you could simplify the statement by using the iterator variable of the ``v-for`` above:

``v-if="article.chapters[i].image"`` -> v-if="chapter.image"
``v-if="article.chapters[i]"`` -> v-if="chapter"